README.md

```diff
@@ -31,11 +31,11 @@ Visit our blockchain explorer to apply for Super Representative, please take [Ru
 
 **Step3.** 
 
-After approval voting, the top 15 Super Representatives by total approval are selected in the testnet stage. The voting of Super Representative is held once every 4 hours. Please take [How to vote](https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/TRON_Blockchain_Explorer_Introduction/How_to_vote.md).
+After approval voting, the top 15 Super Representatives by total approval will be selected in the testnet stage. The voting of Super Representatives is held once every 4 hours. Please read [How to vote](https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/TRON_Blockchain_Explorer_Introduction/How_to_vote.md) for more information.
 
 ## How to become a recommended Super Representative.
 
-To become a recommended Super Representative, candidates must meet nine requirements and submit application to us. We will evaluate the quality and validity of the submitted information, based on the results of which announcements of recommended Super Representatives will be made  on TRON’s official forum (tronsr.org). Please take [A Guide to Recommended Super Representative Application](https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/A_Guide_to_Recommended_Super_Representative_Candidate.md).
+To become a recommended Super Representative, candidates must meet nine requirements and submit application to us. We will evaluate the quality and validity of the submitted information, based on the results of which announcements of recommended Super Representatives will be made  on TRON’s official forum (tronsr.org). Please read [A Guide to Recommended Super Representative Application](https://github.com/tronprotocol/Documentation/blob/master/English_Documentation/A_Guide_to_Recommended_Super_Representative_Candidate.md).
```

Comment:
I can't find any differences.

Reply:
I didn't touch this paragraph. I don't know why GitHub acts like I did